assets/components/cards/render/PictureTextCard.tsx

```diff
@@ -4,19 +4,20 @@ import CardBody from './CardBody';
 import CardFooter from './CardFooter';
 import CardRow from './CardRow';
 import {ICardProps} from '../utils';
+import {IArticle} from 'interfaces';
 
-const getPictureTextPanel = (item: any, picture: any, openItem: any, withPictures: any, cardId: any, listConfig: any) => {
-    const rendition = withPictures && getThumbnailRendition(picture);
+const getPictureTextPanel = (item: any, picture: IArticle | null, openItem: any, withPictures: boolean, cardId: any, listConfig: any) => {
```

Comment:
I believe we could also have types for the other arguments here.
* item: IArticle;
* openItem(IArticle, string): void;
* cardId: string;
* listConfig: IListConfig